colophon.php

```diff
@@ -84,13 +84,38 @@ function team51_credits( $args = array() ) {
 		 *
 		 * @param array $credit_links The associative array of credit links.
 		 * @param array $args         The parsed arguments used by `team51_credits()`.
+		 *
+		 * @return array The filtered associative array of credit links.
 		 */
 		$credit_links = apply_filters( 'team51_credit_links', $credit_links, $args );
 
-		echo implode(
-			esc_html( $args['separator'] ),
-			$credit_links //phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped, this cant be escaped as it runs through a filter
-		);
+		$links = implode( esc_html( $args['separator'] ), $credit_links );
+
+		// If we have an args['wrapper'] set, wrap the output in that.
+		if ( array_key_exists( 'wrapper', $args ) ) {
+			// If there is no class defined, use an empty span.
+			$wrapped_template = '' !== $args['wrapper']
+				? '<span class="%1$s">%2$s</span>'
+				: '<span>%2$s</span>';
```

Comment:
I'd far sooner see this written out, rather than doing assignments, comparisons, and ternaries on a single line.

```suggestion
			$wrapped_template = '<span>%2$s</span>';
			// Otherwise, include it as a class name...
			if ( $args['wrapper'] ) {
				$wrapped_template = '<span class="%1$s">%2$s</span>';
			}
```

Reply:
this has been removed as the args are passed through the shortcode handler by default.